README.md

```diff
@@ -11,6 +16,47 @@ The software consists of theree parts:
 - [xroad-catalog-persistence](xroad-catalog-persistence/README.md)
   * Library used to persist and read persisted data. Used by both of the above.
   
+![X-Road Catalog overview](xroad-catalog.png)
+
+## Environments
+* Internal development environment in the AWS: [AWS xroad-catalog](http://xroad-catalog.i.palveluvayla.com)
+* DVV test: [servicerepotest](https://servicerepotest01.csc.fi)
+* DVV production [servicerepoprod](https://servicerepoprod01.csc.fi)
+
+## Version management
+For versioning, [GitHub Flow](https://guides.github.com/introduction/flow/) is used
+
+* Anything in the master branch is deployable
+* To work on something new, create a branch off from master and given a descriptively name(ie: new-oauth2-scopes)
+* Commit to that branch locally and regularly push your work to the same named branch on the server
+* When you need feedback or help, or you think the branch is ready for merging, open a pull request
+* After someone else has reviewed and signed off on the feature, you can merge it into master
+* Once it is merged and pushed to master, you can and should deploy immediately
+
+## Tools
+Running the X-Road Catalog software requires Linux (Ubuntu or RHEL). As a development environment,  only Ubuntu (>=16.04, 18.04 recommended) is currently supported. It should be possible to use some other Linux distribution for development, but the instructions and helper scripts assume Ubuntu. If you are using some other operating system (e.g. Windows or macOS), the easiest option is to first install Ubuntu 18.04 into a virtual machine.
```

Comment:
Is it not RHEL running on xroad-catalog.i.palveluvayla.com? Or do I remember incorrectly?

Reply:
Updated the text a bit, actually our AWS has been set up with RHEL7, but locally you can use either Ubuntu or RHEL. I am using Ubuntu.